### DISCUSSION TITLE
Information about Outlook

### DISCUSSION THREAD

**Discussion by JCMartell** [MicrosoftDocs/office-developer-client-docs#664] [Q&A]
Feedback similar to @anselm94. I would like to have information on Outlook URI scheme, but this page has been written in 2016 and hasn't been updated yet... I was able to get some information by reverse engineering the app myself, but I think this is something that should be documented by the Office team.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cc9d3afc-5f18-8c4f-1b9c-fd2fe52f49db
* Version Independent ID: a900d01a-0f42-13b8-3289-a2787f7b8499
* Content: [Office URI Schemes](https://docs.microsoft.com/en-us/office/client-developer/office-uri-schemes#feedback)
* Content Source: [docs/office-uri-schemes.md](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/office-uri-schemes.md)
* Product: **office365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by lindalu-MSFT** [comment]
Here's a tech article than might help:
[https://social.technet.microsoft.com/Forums/office/en-US/4ec93560-c9a7-407b-83d5-77ba9736260b/trusting-custom-uri-scheme-from-outlook](https://social.technet.microsoft.com/Forums/office/en-US/4ec93560-c9a7-407b-83d5-77ba9736260b/trusting-custom-uri-scheme-from-outlook)

**Comment by KyNorthstar** [comment]
@lindalu-MSFT That seems unrelated. The question you linked is about the asker wanting to use a URI scheme that they invented, somewhere in the body of the email. They likely want to do this so they can send some email about their app, and have the user click a Magic Link to open some part of their app.

_This_ question is about URI schemes that control Microsoft Outlook, presumably for automation, just like the "Content Source" part of @JCMartell's question says. What this question is looking for would probably look something like this: `ms-outlook:encrypt|u|F10A0D55-0C24-400D-93E9-6D66C9C65E86`. It would request that Outlook perform some action, like opening an email, showing the Calendar, creating a task, toggling encryption on an email, etc.

**Comment by lindalu-MSFT** [comment]
I put in a request to our writers to work with the SMEs and update the content. I was able to repro myself. This is on the backlog and we're waiting to assign resources to it.

**Reply by KyNorthstar** [reply]
@lindalu-MSFT any updates on this?
